### ISSUE TITLE
Compatibility with FancyMenu

### ISSUE THREAD

**Issue by seriousfreezing** [DeDiamondPro/Resourcify#40]
Will it be compatible with FancyMenu? Because there's no way to modify the buttons or access the Resourcify interfaces with FancyMenu.

**Comment by DeDiamondPro** [comment]
Thank you for the suggestion! I will add support for FancyMenu in a future update.

**Comment by DeDiamondPro** [comment]
I made it so the plus and update buttons now act very similar to vanilla buttons in commit ed497ef6e1d5e153ce82474ce037eb67a906f94b, meaning you can trigger them from fancy menu in the same manner as you would trigger a normal vanilla button. This will be included in the next release that will be released after I am done fixing some more issues and finishing doing my tests.

**Comment by DeDiamondPro** [comment]
This has been released in 1.4.1

**Comment by seriousfreezing** [comment]
I tried to pull the update resourcepacks button into a FancyMenu button and I saw that the Widget Locator of that button changes when it is clicked the first time and after the first time.
Is there a fix for this?

**Comment by DeDiamondPro** [comment]
> I tried to pull the update resourcepacks button into a FancyMenu button and I saw that the Widget Locator of that button changes when it is clicked the first time and after the first time. Is there a fix for this?

Hmmm that’s weird, I’ll see if I can find what’s going on there

**Comment by seriousfreezing** [comment]
The fix I've made for now is to write down the id of the button and make an if condition. In the case of the resourcepack screen, I saw 3 different button id/locator values. In the case of the datapack there were 2 or 3.

**Comment by DeDiamondPro** [comment]
@seriousfreezing could you tell me what Minecraft version, mod loader, and fancy menu version you are using since I don't seem to be able to replicate your issue

**Comment by DeDiamondPro** [comment]
@seriousfreezing yeah if you could send me your modpack that would be great, since I am still unable to reproduce the issue. I've created #51 to track this issue.

*Sorry for the slow response, I've been busy and might have forgotten about this*

**Comment by seriousfreezing** [comment]
https://modrinth.com/modpack/bedrock-reloaded/version/17.0.0
Everything's fine. Here's the modpack I mentioned, if you try to mess with the interfaces on the resource pack and datapack screen you'll see this problem.

**Comment by DeDiamondPro** [comment]
@seriousfreezing This should have been fixed in 8c584af0f738960d6a590503e1a4f4a17134aa6c, you can find a prebuilt version here https://github.com/DeDiamondPro/Resourcify/actions/runs/11086026927/artifacts/1991070499 (built by github actions), it would be great if you can confirm this fixes.

Also your modpack is really cool, I didn't know fancymenu could change so much stuff!

**Comment by seriousfreezing** [comment]
Thank you, I really appreciate your compliment! Anyway, I'll try out your fix soon!

**Comment by seriousfreezing** [comment]
I've just tested it, and I confirm 100%, it's been fixed and it's perfect!
